docs/product-fit/enterprise.mdx

```diff
@@ -15,10 +15,10 @@ Our suite of products, including the **Torus Wallet**, **Web3Auth Plug and Play*
 within the SDKs. However, for more intricate customizations and access to pre-constructed UI elements, our advanced offering can be your solution.
 
 - **Torus Wallet**: Tailored to reflect your brand, leveraging the robust features of the Torus Wallet. Enables complete cloning of the website at
-  [app.tor.us](app.tor.us).
+  [app.tor.us](https://app.tor.us).
 
 - **Web3Auth Plug and Play**: Seamlessly integrate into your user experience. The entire flow can be cloned, replicating the functionality of
-  [app.web3auth.io](app.web3auth.io).
+  [app.web3auth.io](https://app.web3auth.io).
```

Comment:
@shahbaz17 This is the only external link that's broken, can we replace this with demo-app.web3auth.io ?